src/js/module/test/comment.test.js

```diff
@@ -0,0 +1,15 @@
+import generateCommentHTML from '../popup/commentshow.js';
+
+test('generateCommentHTML should generate correct comment HTML', () => {
+  const comment = {
+    creation_date: '2023-06-28',
+    username: 'JohnDoe',
+    comment: 'This is a test comment.',
+  };
+
+  const expectedHTML = `<p class="mt-2 comment-p">${comment.creation_date} ${comment.username}: ${comment.comment}</p>`;
+
+  const generatedHTML = generateCommentHTML(comment);
+
+  expect(generatedHTML.trim()).toBe(expectedHTML.trim());
+});
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, according to the previous reviewer, please make sure that you are testing the `Comments counter` feature as stated in the project requirements. At the moment, you're correctly testing that your `generateCommentHTML` is working, but this doesn't give us enough confidence to affirm the same for the counter. Please consider adding this test case! 🤓 🔧